File: 백준/Bronze/1934. 최소공배수/최소공배수.java
import java.io.*;
import java.util.*;

public class Main {

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

        int t = Integer.parseInt(br.readLine());

        while (t-- > 0) {
            String[] strings = br.readLine().split(" ");
            int a = Integer.parseInt(strings[0]);
            int b = Integer.parseInt(strings[1]);
            int gcd_num = gcd(a, b);

            bw.write((a*b/gcd_num)+"\n");
        }

        bw.flush();
    }

    private static int gcd(int a, int b) {
        while (a % b != 0) {
            int temp = a % b;
            a = b;
            b = temp;
        }

        return b;
    }

}

File: 백준/Bronze/1934. 최소공배수/최소공배수.py
import sys


def gcd(a, b):

    while (a % b) != 0:
        temp = a % b
        a = b
        b = temp

    return b


if __name__ == "__main__":

    t = int(sys.stdin.readline())

    for _ in range(t):

        a, b = map(int, sys.stdin.readline().split())

        gcd_num = gcd(a, b)

        print(a*b//gcd_num)
